lib/reporter/util.js

```diff
@@ -28,7 +28,22 @@ function hasFailures(modules) {
 
 function sanitizeOutput(output, delimiter, xml) {
   if (!output || output === '') return '';
-  return _(output.split(/[\r\n]+/)).filter(function (item) {
+  /* eslint-disable no-control-regex */
+  // ([valid-range])|. matches anything but only captures valid-range
+  var validUtf8 = new RegExp('(' + [
```

Comment:
I'm not sure, this was the simplest one I could find that removed all illegal unicode characters. We could have it as one massive line, but that seems worse than having it like this.

If you can find a simpler regex then we can drop this. I think this covers all the astral planes as well.